### PR TITLE
test: Check in all TLS test certs

### PR DESCRIPTION
**Pull request by sunjayBhatia**
Commit Message: test: Check in all TLS test certs
Additional Description:
- Will prevent openssl fork-emulation issues on Windows/msys2 that cause
  test flakiness
- modifies context_impl_test to no longer requires a cert that is
  generated on the fly to expire in 15 days
- Will also help unblock https://github.com/envoyproxy/envoy/pull/13264
- Alternative to https://github.com/envoyproxy/envoy-build-tools/pull/107 as a solution to help with https://github.com/envoyproxy/envoy/pull/13276

Risk Level: Low
Testing: Verified on Linux and Windows locally, modifies unit test setup
Docs Changes: N/A
Release Notes: N/A
Platform Specific Features: N/A

### CODE REVIEW

**Comment by davinci26** [comment]
+1 as it helps with removing test dependencies on bash files.

Test that run on bash script are painful on Windows since you need to go through some hoops to attach a debugger.

**Comment by sunjayBhatia** [comment]
/retest

I specifically ran the format and spelling check locally via Docker so this one is confusing

**Comment by phlax** [comment]
@sunjayBhatia see here https://dev.azure.com/cncf/envoy/_build/results?buildId=55029&view=logs&j=a9c3f16a-7a41-5cb0-a55b-0d11e9c98c50&t=044b2326-1800-5bcd-b44b-584adc5b0b85&l=154

File: test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh [hunk 1]
```diff
@@ -4,23 +4,21 @@
 
 set -e
 
+readonly DEFAULT_VALIDITY_DAYS=${DEFAULT_VALIDITY_DAYS:-730}
```
Comment (lizan) [inline test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh:7]: the permission change of this file is not desired? (changes from `755` to `644`)

**Review by lizan — COMMENTED** [review]
Thanks, this looks good to me.

**Comment by sunjayBhatia** [comment]
> @sunjayBhatia see here https://dev.azure.com/cncf/envoy/_build/results?buildId=55029&view=logs&j=a9c3f16a-7a41-5cb0-a55b-0d11e9c98c50&t=044b2326-1800-5bcd-b44b-584adc5b0b85&l=154

Shellcheck! Ha I didn’t read that far up in the output, thanks @phlax

File: test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh (hunk 1 above)
Comment (sunjayBhatia) [reply test/extensions/transport_sockets/tls/ocsp/test_data/certs.sh:7]: Oh yeah, probably an artifact of developing on windows, will fix

**Comment by phlax** [comment]
i added a ticket as its kinda annoying it doesnt give better output #13709

**Comment by sunjayBhatia** [comment]
huh, all stages are finished but github thinks the pipeline overall is not

**Review by lizan — APPROVED** [review]

**Comment by lizan** [comment]
> huh, all stages are finished but github thinks the pipeline overall is not

sounds like either GH issue or AZP issue, but it is not blocking anyway.

**Pull request merged**